tasks/test.yaml

```diff
@@ -53,11 +58,21 @@ tasks:
 
   - name: validate-packages
     description: "Validate all packages"
+    inputs:
+      # Added to support bypassing passthrough gateway validation on non-k3d distributions.
+      validate_passthrough:
+        description: Whether to validate the passthrough gateway.
+        default: "true"
     # loop through each src/* package and run the validate.yaml task
     actions:
       - cmd: |
-          for package in src/*; do
-            uds run -f ${package}/tasks.yaml validate --no-progress
+          for package in $(ls src); do
+            if [ ! $(echo ${EXCLUDED_PACKAGES} | grep ${package}) ]; then
+              if [ ${package} = "istio" ]; then
+                uds run -f src/${package}/tasks.yaml validate --no-progress --with validate_passthrough=${{ .inputs.validate_passthrough }}
+              fi
+             uds run -f src/${package}/tasks.yaml validate --no-progress
```

Comment:
Should this be an if/else? Otherwise I think we're running the istio validation twice?